plugins/database/src/low_level_odbc.cpp

```diff
@@ -471,27 +468,27 @@ _cllExecSqlNoResult(
         return -1;
     }
 
-    rodsLogSql( sql );
+    log_sql::debug("{}: SQL: [{}]", __func__, sql);
 
     stat = SQLExecDirect( myHstmt, ( unsigned char * )sql, strlen( sql ) );
     switch ( stat ) {
     case SQL_SUCCESS:
-        rodsLogSqlResult( "SUCCESS" );
+        log_sql::debug("{}: result: {}", __func__, "SUCCESS");
         break;
     case SQL_SUCCESS_WITH_INFO:
-        rodsLogSqlResult( "SUCCESS_WITH_INFO" );
+        log_sql::debug("{}: result: {}", __func__, "SUCCESS_WITH_INFO");
         break;
     case SQL_NO_DATA_FOUND:
-        rodsLogSqlResult( "NO_DATA" );
+        log_sql::debug("{}: result: {}", __func__, "NO_DATA");
         break;
     case SQL_ERROR:
-        rodsLogSqlResult( "SQL_ERROR" );
+        log_sql::debug("{}: result: {}", __func__, "SQL_ERROR");
         break;
     case SQL_INVALID_HANDLE:
-        rodsLogSqlResult( "HANDLE_ERROR" );
+        log_sql::debug("{}: result: {}", __func__, "HANDLE_ERROR");
         break;
     default:
-        rodsLogSqlResult( "UNKNOWN" );
+        log_sql::debug("{}: result: {}", __func__, "UNKNOWN");
```

Comment:
Hard-code the result string into the message. No need to pass it as an argument. For example:
```cpp
log_sql::debug("{}: result: SUCCESS", __func__);
```